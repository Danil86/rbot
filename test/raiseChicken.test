do move to entity nearest reachable object * then equip hand egg then activate item done