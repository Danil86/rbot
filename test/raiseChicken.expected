["taskList",[[["move to",["entity nearest reachable object *"]],["equip",["hand","egg"]],["activate item",[]]]]]
